update/Pos/Update/Worker.hs

```diff
@@ -12,6 +12,9 @@ import           Formatting (build, sformat, (%))
 import           Serokell.Util.Text (listJsonIndent)
 import           System.Wlog (logDebug, logInfo)
 
+import           Control.Lens (views)
+
+
```

Comment:
Two blank lines is a bit too much for separation, and `Control.Lens` import should be part of the external imports chunk above, not separate from it.